tests/integration/update_cluster/api_elb_cross_zone/in-v1alpha2.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: kops/v1alpha2
```

Comment:
It's often easier to throw this into "complex" or another existing test (and it's OK to do so, at least when it's a whole test for a single field)